runtime/src/main/java/org/corfudb/runtime/ClusterBootstrapUtil.java

```diff
@@ -0,0 +1,64 @@
+package org.corfudb.runtime;
+
+import org.corfudb.runtime.clients.BaseClient;
+import org.corfudb.runtime.clients.IClientRouter;
+import org.corfudb.runtime.clients.LayoutClient;
+import org.corfudb.runtime.clients.ManagementClient;
+import org.corfudb.runtime.clients.NettyClientRouter;
+import org.corfudb.runtime.view.Layout;
+import org.corfudb.util.NodeLocator;
+import org.corfudb.util.Sleep;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * Utility to bootstrap a cluster.
+ *
+ * <p>Created by zlokhandwala on 1/19/18.
+ */
+@Slf4j
+public class ClusterBootstrapUtil {
+
+    @Getter
+    @Setter
+    private static int retries = 3;
+
+    @Getter
+    @Setter
+    private static long retryTimeout = 1000;
+
+    /**
+     * Bootstraps the given layout.
+     * Attempts to bootstrap each node finite number of times.
+     * If the retries are exhausted, the utility th
+     *
+     * @param layout Layout to bootstrap the cluster.
+     */
+    public static void bootstrapCluster(Layout layout) {
+        for (String server : layout.getAllServers()) {
+            int retry = retries;
+            while (retry-- > 0) {
+                try {
```

Comment:
probably need to log the attempt number too

Reply:
Done.